modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -1781,7 +1781,6 @@ export class Wallet implements IWallet {
       {
         address: params.address,
         amount: params.amount,
-        tokenName: params.tokenName,
```

Comment:
99% sure this will break tokens.

i think the fix you want here is to conditionally set `tokenName`. maybe something like:
```
if (params.tokenName) {
  recipients[0].tokenName = params.tokenName;
}
```